app/mailers/referee_mailer.rb

```diff
@@ -26,11 +21,8 @@ def reference_request_chaser_email(application_form, reference)
     @candidate_name = application_form.full_name
 
     @token = reference.update_token!
-    @reference_link = if FeatureFlag.active?('reference_form')
-                        referee_interface_reference_feedback_url(token: @token)
-                      else
-                        google_form_url_for(@candidate_name, @reference)
-                      end
+    @reference_link = referee_interface_reference_feedback_url(token: @token)
```

Comment:
Maybe you can move `referee_interface_reference_feedback_url(token: @token)` in the view

Reply:
Resolved in c7abaa5